### PR TITLE
1845 docs panda descriptions

### LINKED ISSUE #1845: [Docs?]: Include descriptions within Panda Preset
### Latest version

- [X] I have checked the latest version

### Summary 💡

_No response_

### Motivation 🔦

_No response_

### PR DESCRIPTION
**Pull request by kathydlavoie-ps**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

closes #1845

### CODE REVIEW

File: packages/panda-preset/src/theme/tokens.ts [hunk 1]
```diff
@@ -1,26 +1,77 @@
...
+    description:
+      'z-index value - used to hide an element and its children by stacking it below others',
+    value: -1,
+  },
+  base: { description: 'z-index default value', value: 0 },
 
   // pseudo-elements, borders, etc.
-  decorator: { value: 50 },
+  decorator: {
+    description: 'z-index value - used to position pseudo-elements and borders',
```
Comment (kathydlavoie-ps) [inline packages/panda-preset/src/theme/tokens.ts:30]: @caseybaggz - @bradryanbice and I were discussing what the general use case is for this. It feels hard to put a helpful description.

File: packages/panda-preset/src/theme/tokens.ts [hunk 2]
```diff
@@ -1,26 +1,77 @@
...
-  toast: { value: 1700 },
-  tooltip: { value: 1800 },
+  dropdown: {
+    description:
+      'z-index value - used to set the z-order of dropdown elements and their children to stack above or below other elements according to their z-index value',
+    value: 1000,
+  },
+  sticky: {
+    description:
+      'z-index value - used to set the z-order for elements that stick to their nearest scrolling ancestor.',
```
Comment (caseybaggz) [inline packages/panda-preset/src/theme/tokens.ts:42]:
```suggestion
      'Used for positioning elements that need to stick to their nearest scrolling ancestor in the layer stack',
```

File: packages/panda-preset/src/theme/tokens.ts [hunk 3]
```diff
@@ -1,26 +1,77 @@
...
+    value: 1200,
+  },
+  banner: {
+    description:
+      'z-index value - used to set the z-order of banner elements and their children to stack above or below other elements according to their z-index value',
```
Comment (caseybaggz) [inline packages/panda-preset/src/theme/tokens.ts:47]:
```suggestion
      'Used for positioning any elements that are used like a admonition in the layer stack',
```

File: packages/panda-preset/src/theme/tokens.ts [hunk 4]
```diff
@@ -1,26 +1,77 @@
...
+    value: 1300,
+  },
+  overlay: {
+    description:
+      'z-index value - used to set the z-order of overlay elements and their children to stack above or below other elements according to their z-index value',
```
Comment (caseybaggz) [inline packages/panda-preset/src/theme/tokens.ts:52]:
```suggestion
      'Used for positioning overlay elements in the layer stack',
```

File: packages/panda-preset/src/theme/tokens.ts [hunk 5]
```diff
@@ -1,26 +1,77 @@
...
+    value: 1400,
+  },
+  modal: {
+    description:
+      'z-index value - used to set the z-order of modal elements and their children to stack above or below other elements according to their z-index value',
```
Comment (caseybaggz) [inline packages/panda-preset/src/theme/tokens.ts:57]:
```suggestion
      'Used for positioning modal elements in the layer stack',
```

File: packages/panda-preset/src/theme/tokens.ts [hunk 6]
```diff
@@ -1,26 +1,77 @@
...
+    value: 1500,
+  },
+  popover: {
+    description:
+      'z-index value - used to set the z-order of popover elements and their children to stack above or below other elements according to their z-index value',
```
Comment (caseybaggz) [inline packages/panda-preset/src/theme/tokens.ts:62]:
```suggestion
      'Used for positioning popover elements in the layer stack',
```

File: packages/panda-preset/src/theme/tokens.ts [hunk 7]
```diff
@@ -1,26 +1,77 @@
...
+    value: 1600,
+  },
+  toast: {
+    description:
+      'z-index value - used to set the z-order of toast elements and their children to stack above or below other elements according to their z-index value',
```
Comment (caseybaggz) [inline packages/panda-preset/src/theme/tokens.ts:67]:
```suggestion
      'Used for positioning toast elements in the layer stack',
```

File: packages/panda-preset/src/theme/tokens.ts [hunk 8]
```diff
@@ -1,26 +1,77 @@
...
+    value: 1700,
+  },
+  tooltip: {
+    description:
+      'z-index value - used to set the z-order of tooltip elements and their children to stack above or below other elements according to their z-index value',
```
Comment (caseybaggz) [inline packages/panda-preset/src/theme/tokens.ts:72]:
```suggestion
      'Used for positioning tooltip elements in the layer stack',
```

**Review by caseybaggz — APPROVED** [review]
🎉

**Pull request merged**